### ISSUE TITLE
argparse.ArgumentError: argument -V/--ver: conflicting option string: -V

### ISSUE THREAD

**Issue by qxf2** [qxf2/qxf2-page-object-model#202]
See: 
https://circleci.com/gh/qxf2/qxf2-page-object-model/747

I suspect this is to do with pytest version 5.2.3. For example: https://gitlab.com/pycqa/flake8/issues/594

**Comment by qxf2** [comment]
I ran `pytest -h` and traced the conflicting argument to the reporting plugin that also uses -V. This must have been a recent addition since our tests used to work till recently.

![reporting_plugin_V](https://user-images.githubusercontent.com/4759449/77248763-ebaf8e80-6c61-11ea-8e50-835e6e5a197f.png)

**Comment by qxf2** [comment]
For now, I am removing `-V` from our option parser in conftest.py. We will have to simply use `--ver` when adding a browser version. I have also upgraded our pytest version in requirements so we don't hit the original suspect (pytest version 5.2.2).

Removing the '-V' change requires the following files/places to be updated:
a) conftest.py at:
a1) pytest_addoption()
a2) pytest_generate_tests() where we create the browser list
a3) browser_version() fixture

b) Readme.md in the  `COMMANDS FOR RUNNING TESTS` section

c) utils/Options_parser.py (not needed but made the change to keep things consistent) 

I tested the run by:
a) running pytest with collect-only (no conflicting option error)
b) running pytest locally without the -V or --ver option
c) running pytest using BrowserStack with `pytest -f tests/test_example_form.py -M Y -B Chrome --ver 71 -P Windows -O 10`
